pkg/resource/aws/aws_route.go

```diff
@@ -44,38 +25,4 @@ func initAwsRouteMetaData(resourceSchemaRepository resource.SchemaRepositoryInte
 		val.DeleteIfDefault("instance_owner_id")
 		val.DeleteIfDefault("carrier_gateway_id")
 	})
-	resourceSchemaRepository.SetHumanReadableAttributesFunc(AwsRouteResourceType, func(res *resource.Resource) map[string]string {
-		val := res.Attrs
-		attrs := make(map[string]string)
-		if rtID := val.GetString("route_table_id"); rtID != nil && *rtID != "" {
-			attrs["Table"] = *rtID
-		}
-		if ipv4 := val.GetString("destination_cidr_block"); ipv4 != nil && *ipv4 != "" {
-			attrs["Destination"] = *ipv4
-		}
-		if ipv6 := val.GetString("destination_ipv6_cidr_block"); ipv6 != nil && *ipv6 != "" {
-			attrs["Destination"] = *ipv6
-		}
-		if prefix := val.GetString("destination_prefix_list_id"); prefix != nil && *prefix != "" {
-			attrs["Destination"] = *prefix
-		}
-		return attrs
-	})
-	resourceSchemaRepository.SetFlags(AwsRouteResourceType, resource.FlagDeepMode)
-}
-
-func CalculateRouteID(tableId, CidrBlock, Ipv6CidrBlock, PrefixListId *string) string {
```

Comment:
I think this is used in a route expander middleware nope ? In my mind in the future the enum lib will mostly be under an `internal/` pkg and will only expose List and Refresh methods. We should avoid to add too many coupling, it's IMO OK to have duplicated code here for that kind of stuff.

Reply:
not sure about this one but there are some place where we compute ID and that may be needed for driftctl too when middleware create it too.
I think I checked that it was used in the enum before moving it but I might be wrong for this one...